modules/app.rb

```diff
@@ -7,9 +7,12 @@ class App
   include RentalOperations
   include PersonOperations
 
+  attr_reader :book, :rentals, :person
+
   def initialize
-    @books = []
-    @people = []
+    @books = load_books
+    @people = load_people
+    @rentals = load_rentals
```

Comment:
- [ ] After the data is saved and exits the app, When I run the app again there is an error occurred while loading the data. So please fix that.

![image](https://user-images.githubusercontent.com/68796232/212089354-de900819-7390-49ab-8556-47930408ff80.png)

Reply:
@Omar-Muhamad  Thanks for the feedback. 
The requested changes have been implemented.🙃